### DISCUSSION TITLE
is it possible to copy messages?

### DISCUSSION THREAD

**Discussion by jhnc** [the-djmaze/snappymail#1547] [Q&A] [answered]
Sorry if I'm being stupid. There is a "move" button but I can't see how to make copies of a message. Is this implemented?

**Comment by the-djmaze** [comment, marked as answer]
Press and hold the `Ctrl` key

**Comment by jhnc** [comment]
Thanks. That's somewhat hard to discover, and I'm not sure how to press and hold any physical keys on my touchscreen phone. I'll open an enhancement issue.
